vllm/v1/core/scheduler.py

```diff
@@ -517,6 +517,7 @@ class NewRequestData:
     prompt_token_ids: List[int]
     prompt: Optional[str]
     mm_inputs: List["MultiModalKwargs"]
+    mm_hashes: List[Optional[str]]
```

Comment:
this should be able to be List[str], no?

Reply:
Ah good catch.